new/LIT6374Confessions.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6374Confessions" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">ለክርስቲያን፡ ሁሉ፡ ተስፋ፡ ኑዛዜ፡ የሚሆን፡ ጥቂት፡ ቃል፡</title>
+                <title xml:lang="am" corresp="#t1" type="normalized">lakrǝstiyān hulu tasfā nuzzāze yammihon ṭǝqit q̄al</title>
+                <title xml:lang="en" corresp="#t1">A short word for all Christians as a confession of hope</title>
+                <title xml:lang="en" corresp="t2">Collection of Biblical Confessions from two monks from Dabra Bezan Monastery </title>
```

Comment:
We write "monastery" small, I think; please check if the name is really "Bezan" ቤዘን. In some scripts (in particular in the recent time), -i and -e are written in such a way that they are not immediately distinguishable, at least for us.

Reply:
Thanks for pointing this out, I also was surprised at the spelling of Bezan. Looking at this image, I think it is clear that the third form vowel is shown quite distinctively from the fifth form. 
![EMIP02259_Weiner260_037](https://user-images.githubusercontent.com/39619238/107504010-9a1e3400-6b92-11eb-8c20-16f66ea7dafc.jpg)